src/language-server/project/rover/project.ts

```diff
@@ -38,6 +40,10 @@ export interface RoverProjectConfig extends GraphQLProjectConfig {
   capabilities: ClientCapabilities;
 }
 
+const supportedLanguageIds = (
+  process.env.APOLLO_ROVER_LANUGAGE_IDS || "graphql"
```

Comment:
```suggestion
  process.env.APOLLO_ROVER_LANGUAGE_IDS || "graphql"
```

Typo. Not sure if this affects other places, but you may want to do a quick grep of the codebase to check 🙂

Reply:
Good call, thank you!